### PR TITLE
MRG, FIX: make labels in epochs.plot start at 0, fixes issue #7260

### LINKED ISSUE #7260: Minor: epoch counts in epochs.plot() starts with 1 instead of 0
When using `epochs.plot()` to display epochs, e.g. to manually identify bad ones, the epoch numbers displayed on the plot start with 1, not 0, `while epochs.drop()` expects counts to start from 0. 
[I am aware that one can use the interactive plot + annotations to manually reject epochs, but still think it would avoid confusion if the counters match.]

### PR DESCRIPTION
**Pull request by SophieHerbst**
Makes epochs.plot() start at 0 instead of 1 (issue #7260). 
Note: a bug with spyder (#6528) prevents me from testing what happens after closing the interactive plot. I checked that the indices returned for manually selected epochs start with 0, too. 

Closes #7260

### CODE REVIEW

File: mne/viz/epochs.py [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
     ticks = epoch_times + 0.5 * n_times
     ax.set_xticks(ticks)
     ax2.set_xticks(ticks[:n_epochs])
-    labels = list(range(1, len(ticks) + 1))  # epoch numbers
+    labels = list(range(0, len(ticks)))  # epoch numbers
```
Comment (hoechenberger) [inline mne/viz/epochs.py]: This can be simplified to `list(range(len(ticks)))` :) But probably a matter of taste!

File: mne/epochs.py [hunk 2]
```diff
@@ -1252,7 +1252,10 @@ def drop(self, indices, reason='USER', verbose=None):
         keep = np.setdiff1d(np.arange(len(self.events)), try_idx)
         self._getitem(keep, reason, copy=False, drop_event_id=False)
         count = len(try_idx)
-        logger.info('Dropped %d epoch%s' % (count, _pl(count)))
+        logger.info('Dropped %d epoch%s: %s' % (count, _pl(count),
```
Comment (SophieHerbst) [inline mne/epochs.py]: @hoechenberger: how about this part (formatting and array to string) – is there a more efficient way to do it?

File: mne/epochs.py [hunk 3]
```diff
@@ -1252,7 +1252,10 @@ def drop(self, indices, reason='USER', verbose=None):
         keep = np.setdiff1d(np.arange(len(self.events)), try_idx)
         self._getitem(keep, reason, copy=False, drop_event_id=False)
         count = len(try_idx)
-        logger.info('Dropped %d epoch%s' % (count, _pl(count)))
+        logger.info('Dropped %d epoch%s: %s' % (count, _pl(count), 
+                                                np.array2string(np.sort(try_idx), 
+                                                                precision=0, 
+                                                                separator=',')))
```
Comment (SophieHerbst) [inline mne/epochs.py]: @hoechenberger: how about this part (formatting and array to string) – is there a more efficient way to do it?

**Comment by SophieHerbst** [comment]
I will need help with fixing those tests. no idea what they want.

File: mne/epochs.py (hunk 3 above)
Comment (agramfort) [reply mne/epochs.py]: I would have done:

`', '.join(map(str, np.sort(try_idx))`

rather than np. array2string

Comment (hoechenberger) [reply mne/epochs.py]: I've never used `array2string` and I don't really like `map`, so my approach would've been based on a list comprehension:

```python
', '.join([str(i) for i in np.sort(try_idx)])
```

**Comment by SophieHerbst** [comment]
@agramfort I don't know what an .inc file is, nor where to find it.

**Comment by hoechenberger** [comment]
@SophieHerbst 
https://github.com/mne-tools/mne-python/blob/master/doc/changes/latest.inc

File: mne/viz/tests/test_epochs.py [hunk 4]
```diff
@@ -57,7 +57,7 @@ def test_plot_epochs_basic(capsys):
     epochs.info['lowpass'] = 10.  # allow heavy decim during plotting
     fig = epochs.plot(scalings=None, title='Epochs')
     ticks = [x.get_text() for x in fig.axes[0].get_xticklabels()]
-    assert ticks == ['1']
+    assert ticks == ['0']
```
Comment (SophieHerbst) [inline mne/viz/tests/test_epochs.py]: @agramfort is this correct? I don't fully understand what it should do, but it broke the circle-ci.

**Comment by hoechenberger** [comment]
@SophieHerbst You will want to add youself to `doc/changes/names.inc` as well :) You need to spell your name exactly as you did in `latest.inc`, i.e. add sth like:

```
.. _Sophie Herbst: https://github.com/SophieHerbst
```
(but you may also insert a link to a personal website if you prefer)

**Comment by SophieHerbst** [comment]
Done, thanks @hoechenberger!

File: doc/changes/latest.inc [hunk 5]
```diff
@@ -15,6 +15,8 @@ Current (0.20.dev0)
 Changelog
 ~~~~~~~~~
 
+- Improved :func:`mne.plot_evokeds` to label epoch counts starting from 0 `Sophie Herbst`_
```
Comment (larsoner) [inline doc/changes/latest.inc]: You have to refer to these by where they show up in the MNE API docs: https://mne.tools/dev/generated/mne.viz.plot_evoked.html

Also does it change `mne.viz.plot_evoked` or `mne.viz.plot_epochs`?
```suggestion
- Improved :func:`mne.viz.plot_epochs` to label epoch counts starting from 0 `Sophie Herbst`_
```

Comment (SophieHerbst) [reply doc/changes/latest.inc]: Ups, thank you @larsoner . Done.

**Comment by larsoner** [comment]
Thanks @SophieHerbst !

**Pull request merged**